webrtc.html

```diff
@@ -78,9 +78,10 @@ <h2>Terminology</h2>
     <code><dfn data-cite="!HTML51/webappapis.html#errorevent-errorevent">
     ErrorEvent</dfn></code> interface are defined in [[!HTML51]].</p>
     <p>The concepts <dfn data-cite="!HTML51/webappapis.html#queuing">queue a
-    task</dfn>, <dfn data-cite="!HTML51/infrastructure.html#fire">fire a
-    simple event</dfn> and <dfn data-cite="!HTML51/webappapis.html#networking-task-source">networking
+    task</dfn> and <dfn data-cite="!HTML51/webappapis.html#networking-task-source">networking
     task source</dfn> are defined in [[!HTML51]].</p>
+    <p>The concept <dfn data-cite="!DOM#firing-events">fire an
+    event</dfn> is defined in [[!DOM]].</p>
```

Comment:
What is the https:// link for this? I'm assuming this is equivalent to the "does not bubble, etc etc" language that has been removed int his PR.

Reply:
The link is https://dom.spec.whatwg.org/#firing-events.

To my understanding, "does not bubble etc" has no meaning when the target object is not an DOM element, so we do not need that part.

Reply:
It is important (it affects some properties) but the defaults for those are false.

Reply:
@annevk does `does not bubble` and `is not cancelable` have a meaning even when the target object is not in the DOM tree? Currently we have e.g.
```
Firing an ice candidate event named e with an RTCIceCandidate candidate means that an event with the name e, which does not bubble (except where otherwise stated) and is not cancelable (except where otherwise stated), and which uses the RTCPeerConnectionIceEvent interface with the candidate attribute set to the new ICE candidate, MUST be created and dispatched at the given target.
```
in the spec, but I somehow concluded that the "no bubble, not cancelable" is pointless here.

Reply:
As I said, it's observable, via `bubbles` and `cancelable` on https://dom.spec.whatwg.org/#interface-event, but the defaults for these are false (as they should be for what you're doing here too) so you don't have to mention them explicitly. I was just correcting the bit where you said they have no meaning.

Reply:
Thanks for the clarification!